frontend/src/components/matching/MatchingCard.tsx

```diff
@@ -110,6 +114,9 @@ const MatchingCard = () => {
               placeholder="Select a topic"
               selectedKeys={preferences.topics}
               onChange={handleOnSelectionChange}
+              errorMessage={
+                preferences.topics.length == 0 && <span>Topics is required</span>
+              }
             >
               {optionsTopics.map((value) => (
```

Comment:
Oh, I think if the selected topics are long, it may not fit the matching card well, see this example:
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g05/assets/69421979/17bc8a17-1d8d-4e5c-aa12-d32e0d08d6b2)

I will suggest we prevent this by:
1. limit the max topics selectable
2. increase the width of the modal so that it is able to fit the first 5 longest topics / render the topics into 2 or more lines (we can render into at most 5 lines for example if the max topics allowed is 5)